types/amplitude-js/index.d.ts

```diff
@@ -4,7 +4,10 @@
 //                 Dan Manastireanu <https://github.com/danmana>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare module amplitude {
+export = amplitude;
+export as namespace amplitude;
+
+declare namespace amplitude {
```

Comment:
I don't think you need this namespace or the `export = amplitude;` line if the usage you added to the tests is correct. You will just need to `export` the individual types, classes, and functions you want to be accessible and the `export as namespace amplitude` case will handle the global case.

Reply:
So `export = amplitude;` will do the same as removing the wrapping namespace and exporting each type inline? And `export as namespace amplitude` will be the same in either case? I can do that.